mscore/scoreview.cpp

```diff
@@ -2519,8 +2519,11 @@ void ScoreView::textTab(bool back)
             // TODO: for tempo text, mscore->addTempo() could be called
             // but it pre-fills the text
             // would be better to create empty tempo element
-            if (type != ElementType::TEMPO_TEXT)
-                  cmdAddText(defaultTid, tid);
+            if (type != ElementType::TEMPO_TEXT) {
+                  PropertyFlags pf = oe ? oe->propertyFlags(Pid::PLACEMENT) : PropertyFlags::STYLED;
```

Comment:
Thanks for working on this!

I guess you are using a propertyflags parameter instead of calculating a "force" parameter, and then checking the value of the flags within cmdAddText.  Same thing, I guess.  Just wondering if there some advantage to doing it this way.

Reply:
That `PropertyFlags` should be passed to `cmdAddText()` anyway, because the next element has to have to same value for that. So we can simply define that `bool` once we get the `PropertyFlags` inside `cmdAddText()` rather than passing another parameter.

Reply:
Good point!  It's probably possible to figure out the right flags to use in cmdAddText (the Inspector has similar logic), but your way is more direct.

Reply:
:)